### DISCUSSION TITLE
I propose a separate category or label for the new winget configuration feature

### DISCUSSION THREAD

**Discussion by totkeks** [microsoft/winget-cli#3402] [Ideas] [answered]
Came here to find information about the new configuration feature and discuss with others, but had a hard time finding information for the topic.

Would it be possible to have a separate category or label for the configuration feature, so it is easier to find information on it?

**Comment by mdanish-kh** [comment]
There already is one https://github.com/microsoft/winget-cli/labels/Command-Configure

https://github.com/microsoft/winget-cli/issues?q=is%3Aopen+is%3Aissue+label%3ACommand-Configure

**Reply by totkeks** [reply]
Thank you, I have seen that, but I thought it is only for issues. Also I can't apply the labels myself, which would be really helpful on discussions, that are not visited by bots automatically, like issues do.

Since I don't know what privileges are necessary to allow labeling discussions, it might still be a better idea to have a separate category.

Edited for clarity and to sound less German.

**Reply by mdanish-kh** [reply]
Oh okay, so the request is to have a new category and label here in the `Discussions` section. That can be created by a repository maintainer. cc @denelon / @stephengillie for that request

**Reply by denelon** [reply, marked as answer]
I added Configuration as a category.

**Reply by totkeks** [reply]
Thanks very much! I shall make use of it immediately. :-)
